src/components/contribute/carousel/instructions.tsx

```diff
@@ -86,16 +92,29 @@ export const Instructions: React.FC<Props> = ({
             <Error>{getRecordingErrorMessage()}</Error>
         ) : activeClip ? (
             <Message>Smelltu á örina til að spila upptökuna</Message>
-        ) : (
+        ) : contribute.hasPlayedRepeatClip ? (
             <Message>Smelltu á hljóðnemann og lestu setninguna upp</Message>
+        ) : (
+            <Message>Hlustaðu á upptökuna </Message>
         );
     };
 
+    //console.log(contribute.hasPlayedRepeatClip);
     return (
         <InstructionsContainer>
             {isSpeak ? speakInstructions() : verificationInstructions()}
         </InstructionsContainer>
     );
 };
 
-export default Instructions;
+const mapStateToProps = (state: RootState) => {
+    const contribute = state.contribute;
+    //contribute: {hasPlayedRepeatClip},
+    return { contribute };
+
+    // = State
+    //return {hasPlayedRepeatClip};
+};
+export const Instructions = connect(mapStateToProps)(InstructionsFC);
+
+//i have added setHasPlayedRepeatClip to the reducer , state
```

Comment:
commented out code should be removed, comment should be reserved for code explanation, documentation or temporary removed code